cognite/neat/_utils/text.py

```diff
@@ -192,7 +192,7 @@ def replace_non_alphanumeric_with_underscore(text: str) -> str:
     return re.sub(r"\W+", "_", text)
 
 
-def humanize_collection(collection: Collection[Any], /, *, sort: bool = True) -> str:
+def humanize_collection(collection: Collection[Any], /, *, sort: bool = True, bind_word: str = "and") -> str:
```

Comment:
This utils function here, is it only used to "prettify" the logs/warnings? Can we have some more description on what it does?